truss/templates/server/common/schema.py

```diff
@@ -0,0 +1,85 @@
+import inspect
+from typing import AsyncGenerator, Generator, Optional, Union
+
+from pydantic import BaseModel
+
+
+class TrussSchema(BaseModel):
+    input_type: type
+    output_type: Optional[type]
+    supports_streaming: bool
+
+    @classmethod
+    def from_signature(cls, signature: inspect.Signature) -> Optional["TrussSchema"]:
+        """
+        Create a TrussSchema from a function signature if annotated, else returns None
+        """
+        parameter_types = list(signature.parameters.values())
+
+        if len(parameter_types) > 1:
```

Comment:
How should we notify the suer that their predict function is written improperly?